lib/model/rwfolder.go

```diff
@@ -1454,6 +1456,7 @@ func (f *sendReceiveFolder) performFinish(state *sharedPullerState) error {
 			// we have resolved the conflict.
 
 			state.file.Version = state.file.Version.Merge(state.version)
+			f.lastModBy = state.file.ModifiedBy.String()
 			if err = osutil.InWritableDir(f.moveForConflict, state.realName); err != nil {
```

Comment:
Remove the f.lastModBy (as that is racey as hell, if we handle multiple files at the time for example, the attribute might change to some undefined value which makes no sense)

And use:

```
osutil.InWritableDir(func(name string) {
 f.moveForConflict(name, state.ModifiedBy.String())
}, state.realName);
```
and replace the signature of moveForConflict to accept ModifiedBy.